### DISCUSSION TITLE
Coverage Calculator filtering by level or points

### DISCUSSION THREAD

**Discussion by joe-davenport** [MKT-Playbook/Hub#25] [Feature requests]
When sorting DKG “By New Coverage” it would be nice to have an additional filter to know where coverage is weak. 
For example, I have all tracks covered, but would like to see if I have all tracks covered with at least level 6.

**Comment by TitusPosiadly** [comment]
Good idea. This will probably be covered by the new Coverage Tool (WIP). My current idea (besides the actual coverage improvement) is to have a list of cards that will improve the scores on n numbers of tracks sorted by the number of improved tracks. Would this already be what need or where you looking for something else? Would love to hear your input and your ideal feature addition to best solve this.

**Comment by joe-davenport** [comment]
What I'm asking for might be a little more basic. When in the My Cards view, sorting "By New Coverage" there could be a dropdown to select a minimum Level or Points. In this picture example I'm picturing the filters to the left of "Show Karts Only" and I could select "6" and "595" to see which karts are missing that coverage and the sort would be in the order you mentioned from most effective to least. 
![image](https://user-images.githubusercontent.com/19486607/214729754-f28cce53-5711-4bf9-a8f2-3cd103bb499a.jpeg)

**Comment by TitusPosiadly** [comment]
Yeah, I was more thinking, because I am already working on a tool where this would fit in, what functionality/info are you looking for so I can think about a good way this can be integrated into the new tool. But I think I know what you mean. You basically want to know what cards are under a certain threshold, correct? Also why are your images looking like that 😆 (All squared and no rounded corners)

**Reply by joe-davenport** [reply]
My iPhone 14 Pro on iOS 16.3 in Lockdown Mode so some images and text appear default. 

In terms of higher-level coverage, I was trying to find my gaps. Because I have enough items my current coverage sort is blank. So I wanted to introduce a challenge to the sort that would generate a list to try and focus resources to the items to reach minimum level of 6 in my example. 

Looking forward to the new tool and maybe there is already this feature overlap.

**Reply by TitusPosiadly** [reply]
Ah, gotcha. Ok, I will think about a good way to implement this.
